soc/xtensa/intel_adsp/tools/cavstool_client.py

```diff
@@ -115,7 +115,20 @@ def main():
 if args.quiet:
     log.setLevel(logging.WARN)
 
-HOST = args.server_addr
+if args.server_addr:
+    h = args.server_addr.split(':')
+    l = len(h)
+
+    if l == 3:
+        PORT_LOG = int(h[2])
+        PORT_REQ = int(h[1])
+
+    elif l == 2 and l > 3:
```

Comment:
Guess you wanted `or` here, no?

Reply:
Thanks for the correction, it shall be OR here, fixed.